src/chocolatey/StringExtensions.cs

```diff
@@ -91,7 +91,7 @@ public static string to_string(this string input)
         /// </summary>
         /// <param name="input">The input.</param>
         /// <returns></returns>
-        public static SecureString to_secure_string(this string input)
+        public static SecureString ToSecureString(this string input)

```

Comment:
I suppose it could be. I don't see a specific need for it to be, but if you think it's worth explicitly calling out then we could.

Part of the reason the others are named that way is also due to the naming collision with existing methods, which this one doesn't have to contend with.

Reply:
Yip, understood on the need for ToStringSafe etc, but the meaning of that method also covers the null check that is being done within that method, i.e. making it a `Safe` method to call.  I assuming that the same null check is also being done in the `ToSecureString` method, so the `Safe` addition would also make sense.

Reply:
Fair enough :)